Samples/Samples.csproj

```diff
@@ -29,7 +29,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.1" />
-    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.18" /> 
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.19" />
```

Comment:
https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime/. The latest on is 2.3.20